Tests/Marketplace/marketplace_services.py

```diff
@@ -465,6 +466,20 @@ def is_feed_pack(self, yaml_content, yaml_type):
             if yaml_content.get('name').startswith('TIM '):
                 self._is_feed = True
 
+    def is_siem_pack(self, yaml_content, yaml_type):
+        """
+        Checks if an pack is a SIEM pack. If so, updates Pack._is_siem
+
+        Args:
+            yaml_content: The yaml content extracted by yaml.safe_load().
+            yaml_type: The type of object to check.
+        """
+        if yaml_type == 'Integration':
+            if yaml_content.get('isFetchEvents', False) is True:
+                self._is_siem = True
+        if yaml_type in SIEM_RULES_OBJECTS:
+            self._is_siem = True
+
```

Comment:
I think we can remove this function and use the existing `is_feed_pack` function. We can rename it to `add_pack_type_tags` and add this:
```
 if yaml_content.get('isFetchEvents', False) is True:
                self._is_siem = True
```
after the current feed condition:
```
if yaml_content.get('script', {}).get('feed', False) is True:
                self._is_feed = True
```